core/ts/MaciState.ts

```diff
@@ -337,89 +359,191 @@ class Poll {
+                         currentStateLeavesPathElements.unshift(
+                             this.stateTree.genMerklePath(0).pathElements
+                         )
+                         currentBallots.unshift(this.ballots[0].copy())
+                         currentBallotsPathElements.unshift(
+                             this.ballotTree.genMerklePath(0).pathElements
+                         )
+                         // Since the command is invalid, use vote option index 0
+                         currentVoteWeights.unshift(this.ballots[0].votes[0])
+                         // No need to iterate through the entire votes array if the
+                         // remaining elements are 0
+                         let lastIndexToInsert = this.ballots[0].votes.length - 1
+                         while (lastIndexToInsert > 0) {
+                             if (this.ballots[0].votes[lastIndexToInsert] === BigInt(0)) {
+                                 lastIndexToInsert --
+                             } else {
+                                 break
+                             }
+                         }
+                         const vt = new IncrementalQuinTree(
+                             this.treeDepths.voteOptionTreeDepth,
+                             BigInt(0),
+                             5,
+                             hash5,
+                         )
+                         for (let i = 0; i <= lastIndexToInsert; i ++) {
+                             vt.insert(this.ballots[0].votes[i])
+                         }
+                         currentVoteWeightsPathElements.unshift(
+                             vt.genMerklePath(0).pathElements
+                         )
+
+                        // --------------------------------------
+                        // generate topup circuit inputs
+                        let stateIndex = BigInt(message.data[0])
+                        let amount = BigInt(message.data[1])
+
+                        if (
+                            stateIndex >= BigInt(this.ballots.length) ||
+                            stateIndex < BigInt(1) ||
+                            amount < BigInt(1)
+                        ) {
+                            throw Error("no-op")
+                            return {}
+                        }
+                        topupStateIndexes.unshift(stateIndex)
+                        topupAmounts.unshift(amount)
+                        topupStateLeaves.unshift(this.stateLeaves[Number(stateIndex)].copy())
+                        topupStateLeavesPathElements.unshift(
+                            this.stateTree.genMerklePath(Number(stateIndex)).pathElements
+                        )
+
+                        const newStateLeaf = this.stateLeaves[Number(stateIndex)].copy()
+                        newStateLeaf.voiceCreditBalance = BigInt(newStateLeaf.voiceCreditBalance) + BigInt(amount)
```

Comment:
It causes [build failure]((https://github.com/privacy-scaling-explorations/maci/runs/6802556146?check_suite_focus=true#step:6:60)):
```
Error: ts/MaciState.ts(513,66): error TS2345: Argument of type 'BigInt' is not assignable to parameter of type 'string | number | bigint | boolean'.
```

 Seems its the same case as previous error (see [this](https://github.com/privacy-scaling-explorations/maci/pull/406#discussion_r851608061)).